docs/guides/admin/docs/modules/studio.md

```diff
@@ -14,6 +14,17 @@ The path `/studio` is accessible by users with the role `ROLE_ADMIN` or `ROLE_ST
 
 The preferred way to let your users access Studio is via LTI. Remember to configure your LTI users to have the role `ROLE_STUDIO` so that they can access Studio and all APIs used by Studio.
 
+This can be done for example by configuring the `LtiLaunchAuthenticationHandler` in `etc/org.opencastproject.security.lti.LtiLaunchAuthenticationHandler.cfg`. At the bottom of this file you see the two options `lti.custom_role_name` and `lti.custom_roles`. The former should be set to an LTI role that you want to give access to Studio to, and the latter is a comma-separated list of Opencast roles that users with this LTI role should get. This list has to at least include `ROLE_STUDIO`.
```

Comment:
I would add commas around "for example" in the first sentence. 

"access to Studio to" -> "Studio access to"?

Reply:
> I would add commas around "for example" in the first sentence.
> 
Apparently you are right. x)

> "access to Studio to" -> "Studio access to"?

I don't like that, but reading it again, I don't like mine anymore either. :sweat_smile: 

How about "an LTI role that you want to be able to access Studio"?

Reply:
"an LTI role that you want to be able to access Studio" -> mhh, not completely happy about that. "a role that is supposed to have access to studio"? But yeah, nit picking at this point. Don't block on this mini concern.

Reply:
I can live with that. Changed.